### PR TITLE
Easter egg if the hug target is the bot

### PR DESCRIPTION
**Pull request by MoonlightCapital**
# Please check the following boxes

- [x] I've read and agree to the [Contribution Guidelines](https://github.com/Dougley/MBv2/blob/master/.github/CONTRIBUTING.md) and to the [Code of Conduct](https://github.com/Dougley/MBv2/blob/master/.github/CODE_OF_CONDUCT.md)
- [x] I've checked that I didn't commit sensitive information like tokens or other login information
- [x] I haven't hardcoded any IDs or anything else that shouldn't be (with the exception being permissions)
- [x] I tested my code and I verified it's working to the best of my ability
- [x] I checked if my code doesn't violate the styleguide with `npm test`

# Describe your pull request

### CODE REVIEW

File: src/commands/hug.js [hunk 1]
```diff
@@ -28,7 +30,11 @@ module.exports = {
     let id = suffix.match(/<@!?([0-9]*)>/)[1]
     if (id === msg.author.id) return msg.channel.createMessage("Can't execute this action on yourself")
     const user = bot.users.get(id) || bot.getRESTUser(id)
-    const random = templates[Math.floor(Math.random() * templates.length)].replace(/{s}/g, msg.author.username).replace(/{r}/g, user.username)
+    let random = templates[Math.floor(Math.random() * templates.length)].replace(/{s}/g, msg.author.username).replace(/{r}/g, user.username)
```
Comment (Dougley) [inline src/commands/hug.js]: Why did you change this to `let`?

Comment (MoonlightCapital) [reply src/commands/hug.js]: See line 36, so if the target is the same as the bot, `random` will be set to the srting used for that occasion.
